fundamentals/examples.tex

```diff
@@ -151,79 +151,111 @@ \subsection{Simulations}
+\Cyclus flexibility allows this transition to be examined.  The facilities
+used in these simulations are:
+
+\begin{itemize}
+
+    \item \textbf{Reactor} (\Class{cycamore::Reactor}): This is a reactor
+        model that requests any number of input fuel types and uses static
+        compositions to transmute them to after they are burnt and discharged
+        from the core.  Due to the static spent fuel compositions spent fuel
+        compositions become "incorrect" as the input fuel diverges from the
+        fixed, expected input composition. The reactor was configured to model
+        a light water reactor with a 3 batch core (20,000 kg HM per batch)
+        operating on an 18 month cycle with a 2 month refuel time.  The
+        reactor was also configured to take in either enriched UOX fuel or
+        recycled MOX fuel - preferring the latter if available.
+
+    \item \textbf{Recycled Fuel Fabrication} (\Class{cycamore::Fab}): This
+        facility requests depleted uranium and separated fissile material and
+        mixes them together into recycled MOX fuel that it offers to
+        requesters.  The streams are mixed using a D-factors approach
+        \TODO{add D-factors citation} in an attempt to match simple neutronics
+        properties of an ideal target fuel composition.
+
+    \item \textbf{Separations} (\Class{cycamore::Separations}): This facility
+        takes in all kinds of spent fuel and separates it into plutonium and
+        waste streams with some efficiency (0.99 was used for these
+        simulations).  Up to 60,000 kg of Pu can be separated per month.
+
+    \item \textbf{Repository} (\Class{cycamore::Sink}): This is an
+        infinite-capacity facility that can take in all types of material
+        including separations waste streams and spent reactor fuel.
+
+    \item \textbf{UOX Fabrication} (\Class{cycamore::Source}): This facility
+        provides enriched UOX fuel as requested with infinite capacity.
+
+    \item \textbf{DU Source} (\Class{cycamore::Source}): This facility provides
+        depleted uranium as requested with infinite capacity.
+
+\end{itemize}
 
-The \Class{Source} and \Class{Sink} archetypes from \Cycamore have been used as a
-source of fresh, enriched uranium and a repository, respectively. A simple
-custom recycled fuel fabrication facility was also used, which mixes fissile 
-and fertile streams to match target fuel compositions as closely
-as possible.  The method used for this matching is approximately the same as
-that used by the \gls{COSI} fuel cycle simulator \TODO{cite COSI equivalence
-method}. The material flows for this simulation are shown below in Figure
-\ref{fig:flowmodopen}.
+For the once through case, the \Class{Reactor} was set to offer all spent fuel
+as waste.  For the modified open case, the \Class{Reactor} offerred spent UOX
+into a spent fuel market where it is preferrentially sent to separations, but
```

Comment:
Does this seem like enough, the right kind of focus?  Do you have any ideas
for how to recast it to make it a more compelling show of Cyclus'
capability?

On Thu, Feb 5, 2015 at 9:33 PM, Katy Huff <notifications@github.com> wrote:

> In fundamentals/examples.tex
> <https://github.com/cyclus/fundamentals-paper/pull/72#discussion_r24220744>
> :
>
> >
> > -The \Class{Source} and \Class{Sink} archetypes from \Cycamore have been used as a
> > -source of fresh, enriched uranium and a repository, respectively. A simple
> > -custom recycled fuel fabrication facility was also used, which mixes fissile
> > -and fertile streams to match target fuel compositions as closely
> > -as possible.  The method used for this matching is approximately the same as
> > -that used by the \gls{COSI} fuel cycle simulator \TODO{cite COSI equivalence
> > -method}. The material flows for this simulation are shown below in Figure
> > -\ref{fig:flowmodopen}.
> > +For the once through case, the \Class{Reactor} was set to offer all spent fuel
> > +as waste.  For the modified open case, the \Class{Reactor} offerred spent UOX
> > +into a spent fuel market where it is preferrentially sent to separations, but
>
> preferentially
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/fundamentals-paper/pull/72/files#r24220744>.
>